index.html

```diff
@@ -22,37 +22,50 @@
 
         </div>
     </header>
-    <section>
+
+    <!-- Intro section -->
+    <section id="header-section">
         <div class="header">
-            <div class="header-content">
-                <h1 class="header-title">Hey there. <br>I am Alex.</h1>
-                <p class="header-sub-title">I'm a software developer</p>
-                <p class="supporting-text">
-                    I can help you build a product,<br>feature or website Look through <br> some of my work and
-                    experience! <br> If you
-                    like what you see and have <br> a project you need coded,<br>
-                    don’t hesitate to contact me.
-                </p>
-                <ul class="social-icons">
-                    <li><a href="#"><img src="./assets/icons/twitter.svg" alt="Twitter icon"></a></li>
-                    <li><a href="#"><img src="./assets/icons/linkedin.svg" alt="LinkedIn icon"></a></li>
-                    <li><a href="#"><img src="./assets/icons/medium.svg" alt="Medium icon"></a></li>
-                    <li><a href="#"><img src="./assets/icons/github.svg" alt="GitHub icon"></a></li>
-                    <li><a href="#"><img src="./assets/icons/angellist.svg" alt="AngelList icon"></a></li>
-
-
-                </ul>
-                <div class="chevron-icon">
-                    <a href="#"><img src="./assets/icons/chevron.svg" alt="Chevron icon"></a>
-                </div>
+            <div class="header-title">
+                <h1><span>Hey there.</span><br><span>I'm Alex.</span></h1>
+                <h2>I'm a software<br>developer</h2>
+
+            </div>
+
+            <p class="supporting-text">I can help you build a product,<br>some of my work and
+                experience!<br>
+                If you like what you see and have
+                <br>a project you need coded,<br>don’t hesitate to contact me.
+            </p>
```

Comment:
- [OPTIONAL] I am going to make this optional since it wasn't pointed out by the previous reviewer.  You have done a great job so far but I think there is one improvement you need to make to your code base before moving to the next milestone, :+1: :+1: :+1: 
- - Currently the background images on the Figma design don't show on your website, I can see you actually added background images in your CSS but they are not correctly displaying in the browser, I'll suggest you look into this, as its part of the requirement of making your project look exactly link the Figma design and also it gives the UI a very nice and professional look. :smile: :+1: 
- Figma design:
![Selection_027](https://user-images.githubusercontent.com/107632780/235519608-0af70f9d-f078-4b29-aef5-3dabff1400b3.png)

- Your Current design:
![Selection_028](https://user-images.githubusercontent.com/107632780/235519730-48681358-691e-427f-a50f-bed0fdba9a84.png)